src/openms/source/FORMAT/HANDLERS/MzXMLHandler.cpp

```diff
@@ -279,7 +279,7 @@ namespace OpenMS
         }
 
         // Add a new spectrum, initialize and set MS level and RT
-        spectrum_data_.resize(spectrum_data_.size() + 1); // TODO !!
+        spectrum_data_.resize(spectrum_data_.size() + 1);
```

Comment:
I see its basically a `push_back` of an empty spectrum done in a "smart" way. But it seems modern compilers are smart enough that it doesnt make a difference: http://stackoverflow.com/questions/1902832/resize-versus-push-back-in-stdvector-does-it-avoid-an-unnecessary-copy-assig

still, is fine with me